shell/platform/windows/win32_dpi_helper.cc

```diff
@@ -13,60 +15,57 @@ bool AssignProcAddress(HMODULE comBaseModule, const char* name, T*& outProc) {
 }  // namespace
 
 Win32DpiHelper::Win32DpiHelper() {
-  // TODO ensure that this helper works correctly on downlevel builds.
   user32_module_ = LoadLibraryA("User32.dll");
   if (user32_module_ == nullptr) {
     return;
   }
 
-  if (!AssignProcAddress(user32_module_, "EnableNonClientDpiScaling",
-                         enable_non_client_dpi_scaling_)) {
+  if (AssignProcAddress(user32_module_, "GetDpiForWindow",
+                        get_dpi_for_window_)) {
+    dpi_for_window_supported_ = true;
     return;
   }
 
-  if (!AssignProcAddress(user32_module_, "GetDpiForWindow",
-                         get_dpi_for_window_)) {
+  shlib_module_ = LoadLibraryA("Shcore.dll");
+  if (shlib_module_ == nullptr) {
     return;
   }
 
-  if (!AssignProcAddress(user32_module_, "SetProcessDpiAwarenessContext",
-                         set_process_dpi_awareness_context_)) {
-    return;
```

Comment:
SetProcessDpiAwarenessContext is the most recent version of the API.. any reason why you chose not to use this if available?

Reply:
I just removed it since it's not being used anywhere, and we would want the dpi awareness to be set by the runner app, in the manifest anyway. Is there any reason to keep it?

Reply:
OK, yes good point :-)